src/test/java/com/mysql/cj/jdbc/ha/plugins/failover/FailoverConnectionPluginTest.java

```diff
@@ -520,14 +512,13 @@ public void testRdsProxy() throws SQLException {
   public void testRdsReaderCluster() throws SQLException {
     final ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
     final String url =
-        "jdbc:mysql:aws://my-cluster-name.cluster-XYZ.us-east-2.rds.amazonaws.com";
+        "jdbc:mysql:aws://my-cluster-name.cluster-ro-XYZ.us-east-2.rds.amazonaws.com";
```

Comment:
Seemed like either the name of the test or the url had to change for this test; the original test name indicated a reader cluster but the original url indicated a writer cluster. I opted for changing the url but let me know if I should instead change the name